### PR TITLE
Removal of redundant code

### PR DESCRIPTION
**Pull request by andrijaJ01**

### CODE REVIEW

File: src/components/Tutorial/QuestionsTab/index.jsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ const QuestionTabsSection = () => {
       </Row>
       */}
 
-      <Row >
+      <Row classname='w-100'>
         <Col className='d-flex'>
```
Comment (Mateja3m) [inline src/components/Tutorial/QuestionsTab/index.jsx:86]: Where is closing tag for Col

File: src/components/Tutorial/index.jsx [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
 const TutorialSection = () => {
+  const [activeRow, setActiveRow] = useState('user-site');
+
+  const handleRowChange = (rowId) => {
+    setActiveRow(rowId);
+  };
+
   return (
-    <section>
+    <section className='w-80'>
```
Comment (Mateja3m) [inline src/components/Tutorial/index.jsx:16]: Do not give width within component, you giving widht of the component in the parent, where component is shown.

File: src/components/Tutorial/index.jsx [hunk 3]
```diff
@@ -16,15 +23,26 @@ const TutorialSection = () => {
...
+              <Link to="#project-site" className="lh-1em position-relative text-dark">Project site</Link>
             </Col>
           </Row>
         </Col>
       </Row>
+      {/* Render QuestionTabsSection based on activeRow state */}
+      {activeRow === 'user-site' && (
+        <Row>
+          <QuestionTabsSection />
+        </Row>
```
Comment (Mateja3m) [inline src/components/Tutorial/index.jsx:39]: Commented code remove from component.
You have this comment too @andrijaJ01

File: src/components/Tutorial/QuestionsTab/index.jsx (hunk 1 above)
Comment (andrijaJ01) [reply src/components/Tutorial/QuestionsTab/index.jsx:86]: line 97

File: src/components/Tutorial/index.jsx (hunk 2 above)
Comment (andrijaJ01) [reply src/components/Tutorial/index.jsx:16]: I'll make sure to address this concern by ensuring that width specifications are handled with more care

File: src/components/Tutorial/index.jsx (hunk 3 above)
Comment (andrijaJ01) [reply src/components/Tutorial/index.jsx:39]: on it

**Pull request merged**